platform/rct/hal/usart_stm32/usart_device_stm32.c

```diff
@@ -33,30 +33,41 @@
 #include "stm32f30x_rcc.h"
 #include "stm32f30x_usart.h"
 #include "task.h"
+#include "timers.h"
 #include "usart_device.h"
 
-#define UART_RX_QUEUE_LEN 	256
-#define UART_TX_QUEUE_LEN 	64
-#define GPS_BUFFER_SIZE		132
-
-#define UART_WIRELESS_IRQ_PRIORITY 	7
+#define DEFAULT_GPS_BAUD_RATE		115200
```

Comment:
It does not and arguable can go away.  But I have lots of cleaning of this UART code to do before that is ready to go away.  When I add support for lazy init, then this will die.